components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -203,6 +203,9 @@ be lost if SCREAM_HACK_XML is not enabled.
         ${DIN_LOC_ROOT}/atm/scream/tables/vn_table_vals.dat8,
         ${DIN_LOC_ROOT}/atm/scream/tables/vm_table_vals.dat8
       </tables>
+      <p3_pre_autoconversion_factor type="real" doc="P3 pre_autoconversion_factor">1350.0</p3_pre_autoconversion_factor>
```

Comment:
Can we make the doc string a little bit more explanatory? So far they seem to just restate the name of the param, but I have no idea what "spa_to_nc" means...

I know we're in a rush to get these params exposed for the autotuning work, so I understand if you want to punt. If you want to punt, maybe some other dev with P3 knowledge could help? @AaronDonahue maybe?

Reply:
i do not know what these parameters mean either, if anyone wants to add another column for descriptions in https://acme-climate.atlassian.net/wiki/spaces/EAMXX/pages/4055498787/Auto-Calibration+Tuning+Parameters i will put them in the pr. thanks!